actions/22x22-light/.directory

```diff
@@ -0,0 +1,3 @@
+[Dolphin]
```

Comment:
Please do not commit `.directory` files, or other extraneous filesystem metadata.

Reply:
Any `.directory` files you have committed will need to be removed